hieradata_aws/class/integration/ckan_db_admin.yaml

```
@@ -0,0 +1,49 @@
+govuk_env_sync::tasks:
+  "pull_ckan_production_daily":
+    ensure: "present"
+    hour: "0"
+    minute: "0"
+    action: "pull"
+    dbms: "postgresql"
+    storagebackend: "s3"
+    database: "ckan_production"
+    database_hostname: "ckan-postgresql"
+    temppath: "/tmp/ckan_production"
+    url: "govuk-production-database-backups"
+    path: "ckan-postgresql"
```

```suggestion
    path: "postgresql-postgresql"
```

There won't be anything in s3://govuk-production-database-backups/ckan-postgresql until production backups start pushing there, so we need to pull from the existing backup source: s3://govuk-production-database-backups/postgresql-primary

I have a feeling this might be a problem we have in all the new db_admins to fix. 

This same comment applies to lots of places.

---

Should this be `postgresql-backend` @kevindew? I've just logged into AWS and looked in the govuk-production-database-backups S3 and can't find a `postgresql-postgresql` path.

---

Oh gosh sorry! I'm in a meeting but I think it's postgresql-primary not postgresql-backend but worth a check.

---

I edited the comment to attach a screenshot from AWS - we'll continue on the basis that it's `postgresql-backend` 👍

---

This has been done now. I've only updated the `pull` tasks, as the `push` ones are 1) commented out anyway, and 2) probably _will_ want to pull from the new path, which can be tackled later.